test/function_compilation.jl

```diff
@@ -218,11 +218,26 @@ end
             slowtime = @timed select(df_wide, AsTable(:) => ByRow(t -> (sum ∘ skipmissing)(t)) => :y)
 
             (slowtime[2] > fasttime[2]) || @warn("Slow compilation")
+
+            @test @select(df, :y = f(g(:a, :b))).y == [3]
+
+            fasttime = @timed @select(df, :y = f(g(:a, :b)))
+            slowtime = @timed select(df, [:a, :b] => ((a, b) -> f(g(a, b))) => :y )
+            (slowtime[2] > fasttime[2]) || @warn("Slow compilation")
+
+            fasttime = @timed @rselect df_wide :y = sum(skipmissing(AsTable(:)))
```

Comment:
I would add also correctness tests (not only timing).
Also I think it would be good to add more tests of different column selectors.

Reply:
Add more correctness test and with different column selectors.